Makefile

```diff
@@ -2,41 +2,42 @@
 
 .DEFAULT_GOAL := help
 
+NPM_BIN = $(shell npm bin)
+
+# Generates a help message. Borrowed from https://github.com/pydanny/cookiecutter-djangopackage.
 help: ## display this help message
 	@echo "Please use \`make <target>' where <target> is one of"
-	@perl -nle'print $& if m{^[a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
+	@perl -nle'print $& if m{^[\.a-zA-Z_-]+:.*?## .*$$}' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-25s\033[0m %s\n", $$1, $$2}'
 
 install: ## install requirements
 	npm install
 
 clean: ## remove build artifacts
-	rm -rf css/
-	rm -rf samples/css/
-	rm -rf samples/edx-bootstrap/
+	rm -rf {css,samples/{css,edx-bootstrap}}
 
-samples: clean ## build the samples
-	mkdir samples/edx-bootstrap
+build.samples: clean ## build the samples
+	$(eval $@_INCLUDE = --include-path samples --include-path node_modules)
+	mkdir -p samples/{edx-bootstrap,css/{edx,open-edx}}
 	cp -r sass/ samples/edx-bootstrap/sass
-	mkdir samples/css
-	mkdir samples/css/edx
-	mkdir samples/css/open-edx
-	./node_modules/node-sass/bin/node-sass samples/edx/sass --output samples/edx/css --include-path samples --include-path node_modules
-	./node_modules/node-sass/bin/node-sass samples/open-edx/sass --output samples/open-edx/css --include-path samples --include-path node_modules
-	./node_modules/.bin/opn samples/index.html
+	$(NPM_BIN)/node-sass samples/edx/sass --output samples/edx/css $($@_INCLUDE)
+	$(NPM_BIN)/node-sass samples/open-edx/sass --output samples/open-edx/css $($@_INCLUDE)
 
-build: clean ## build the npm package
+samples: clean build.samples ## build and show the samples
+	$(NPM_BIN)/opn samples/index.html
 
-preview: samples ## build the preview site
-	aws s3 sync samples s3://${S3_PREVIEW_DOMAIN}/$(shell git rev-parse --abbrev-ref HEAD)
-	@echo Preview generated to http://${S3_PREVIEW_DOMAIN}/$(shell git rev-parse --abbrev-ref HEAD)
+preview: build.samples ## build the preview site
```

Comment:
@andy-armstrong Can you test this change to make sure it works. Thanks.

Reply:
👍 works nicely, thanks!